src/utils/AwsCliUtil.js

```diff
@@ -25,8 +24,17 @@ module.exports = {
     async configureWithSamlAssertion(roleArn, principalArn, samlAssertion, duration) {
         let credentials = await assumeRoleWithSaml(roleArn, principalArn, samlAssertion, duration);
         await writeConfigFile(credentials);
+    },
+
+    async checkIfConfigFileExists() {
+        const configFilePath = os.homedir() + path.sep + '.aws' + path.sep + 'config';
```

Comment:
This is the second 'configFilePath' we should make a constant var that refers to the AwsCli directory where all the config/credentials files are. And reference them here.